docs/docs/alephium/entrypoints.md

```diff
@@ -0,0 +1,652 @@
+
+#### Input parameters
+
+| Name             | Type        | Description                                                                             |
+| ---------------- | ----------- | --------------------------------------------------------------------------------------- |
+| poolKey         | PoolKey     | Pool key of the pool on which you wish to perform the swap.                             |
+| xToY           | Bool        | Specifies the direction of the swap.                                                    |
+| amount           | U256 | Amount of tokens you want to receive or give.                                           |
+| byAmountIn     | Bool        | Indicates whether the entered amount represents the tokens you wish to receive or give. |
+| sqrtPriceLimit | U256   | If the swap achieves this square root of the price, it will be canceled.                |
+
+#### Output parameters
+
+| Type        | Description                                                                                                                                                                         |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| QuoteResult | A struct containing amount of tokens received, amount of tokens given and square root of price after the simulated swap. |
+
+#### Errors
+
+| Code                | Description                                                                                                         |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------- |
+| `ZeroAmount`      | Fails if the user attempts to perform a swap with zero amounts.                                                     |
+| `WrongPriceLimit` | Fails if the square root of price or price limit is incorrect. |
+| `PriceLimitReached` | Fails if the price has reached the specified price limit (or price associated with specified square root of price). |
+| `TickLimitReached` | Fails if the tick index has reached the global tick limit. |
+| `NoGainSwap`        | Fails if the user would receive zero tokens.                                                                        |
+| `PoolNotFound`      | Fails if pool does not exist.                                                                                       |
+
+## Tick
+
+### Get tick
+
+```rust
+pub fn getTick(poolKey: PoolKey, index: I256) -> (Bool, Tick);
+```
+
+Retrieves information about a tick at a specified index.
+
+#### Input parameters
+
+| Name  | Type    | Description                                      |
+| ----- | ------- | ------------------------------------------------ |
+| poolKey   | PoolKey | A unique key that identifies the specified pool. |
+| index | I256     | The tick index in the tickmap.                   |
```

Comment:
This is not the tick index in the tickmap, but the tick index (do not write this in the docs: this is tick index according to the formula `price = 1.0001^index`)